src/graph_notebook/notebooks/04-Machine-Learning/Neptune-ML-04-Introduction-to-Edge-Classification-Gremlin.ipynb

```diff
@@ -442,7 +442,9 @@
     "--job-id {training_job_name} \n",
     "--data-processing-id {training_job_name} \n",
     "--instance-type ml.p3.2xlarge\n",
-    "--s3-output-uri {str(s3_bucket_uri)}/training \"\"\""
+    "--s3-output-uri {str(s3_bucket_uri)}/training\n",
+    "--max-hpo-number 2\n",
```

Comment:
Does this mean that these parameters are now required?

Reply:
Correct, this is to main consistency with the [Modeltrainining API](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-api-modeltraining.html), which has `maxHPONumberOfTrainingJobs` and `maxHPOParallelTrainingJobs` as non-optional parameters.